src/app/components/LinkedData/__snapshots__/index.test.jsx.snap

```diff
@@ -8,7 +8,7 @@ exports[`LinkData should correctly render metadata for links 1`] = `
   <script
     type="application/ld+json"
   >
-    {"@context":"http://schema.org","@type":"article","url":"https://www.bbc.com/news/articles/c9rpqy7pmypo","publisher":{"@type":"NewsMediaOrganization","name":"BBC News","publishingPrinciples":"https://www.bbc.com/news/help-41670342","logo":{"@type":"ImageObject","width":1024,"height":576,"url":"https://www.bbc.com/news/special/2015/newsspec_10857/bbc_news_logo.png"}},"datePublished":"2018-10-10T16:19:31.344Z","dateModified":"2019-02-26T11:25:10.555Z","headline":"Royal wedding 2018: Bouquet laid on tomb of unknown warrior","image":{"@type":"ImageObject","width":1024,"height":576,"url":"https://www.bbc.com/news/special/2015/newsspec_10857/bbc_news_logo.png"},"thumbnailUrl":"https://www.bbc.com/news/special/2015/newsspec_10857/bbc_news_logo.png","author":{"@type":"NewsMediaOrganization","name":"BBC News","logo":{"@type":"ImageObject","width":1024,"height":576,"url":"https://www.bbc.com/news/special/2015/newsspec_10857/bbc_news_logo.png"},"noBylinesPolicy":"https://www.bbc.com/news/help-41670342#authorexpertise"},"about":[{"@type":"Person","name":"Duchess of Sussex","alternateName":"Meghan Markle","sameAs":["http://dbpedia.org/resource/Meghan_Markle"]}],"mainEntityOfPage":{"@type":"WebPage","@id":"https://www.bbc.com/news/articles/c9rpqy7pmypo","name":"Royal wedding 2018: Bouquet laid on tomb of unknown warrior"}}
+    {"@context":"http://schema.org","@type":"article","url":"https://www.bbc.com/news/articles/c9rpqy7pmypo","publisher":{"@type":"NewsMediaOrganization","name":"BBC News","publishingPrinciples":"https://www.bbc.com/news/help-41670342","logo":{"@type":"ImageObject","width":1024,"height":576,"url":"https://www.bbc.com/news/special/2015/newsspec_10857/bbc_news_logo.png"}},"image":{"@type":"ImageObject","width":1024,"height":576,"url":"https://www.bbc.com/news/special/2015/newsspec_10857/bbc_news_logo.png"},"thumbnailUrl":"https://www.bbc.com/news/special/2015/newsspec_10857/bbc_news_logo.png","mainEntityOfPage":{"@type":"WebPage","@id":"https://www.bbc.com/news/articles/c9rpqy7pmypo","name":"Royal wedding 2018: Bouquet laid on tomb of unknown warrior"},"headline":"Royal wedding 2018: Bouquet laid on tomb of unknown warrior","datePublished":"2018-10-10T16:19:31.344Z","dateModified":"2019-02-26T11:25:10.555Z","author":{"@type":"NewsMediaOrganization","name":"BBC News","logo":{"@type":"ImageObject","width":1024,"height":576,"url":"https://www.bbc.com/news/special/2015/newsspec_10857/bbc_news_logo.png"},"noBylinesPolicy":"https://www.bbc.com/news/help-41670342#authorexpertise"},"about":[{"@type":"Person","name":"Duchess of Sussex","alternateName":"Meghan Markle","sameAs":["http://dbpedia.org/resource/Meghan_Markle"]}]}
   </script>
```

Comment:
I triple checked this and all the tags are coming through as expected, but they're just in a different order.